Sources/AsyncQueue/ActorQueue.swift

```diff
@@ -154,7 +154,7 @@ public final class ActorQueue<ActorType: Actor>: @unchecked Sendable {
 }
 
 extension Actor {
-    func suspendUntilStarted(_ task: @escaping @Sendable (isolated Self) async -> Void) async {
+    fileprivate func suspendUntilStarted(_ task: @escaping @Sendable (isolated Self) async -> Void) async {
```

Comment:
not a required change at all but since I don't have an expectation this method to be used outside of this file I might as well lock it down.